snippets/markdown/image/tensorflowjs/javascript.md

```diff
@@ -1,48 +1,56 @@
 ```html
-<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs@1.1.2/dist/tf.min.js"></script>
-<script src="https://cdn.jsdelivr.net/npm/@teachablemachine/image@0.6/dist/teachablemachine-image.min.js"></script>
+<div>Teachable Machine Image Model</div>
+<button type='button' onclick='init()'>Start</button>
+<div id='webcam-container'></div>
+<div id='label-container'></div>
+<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs@1.3.1/dist/tf.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/@teachablemachine/image@0.8.3/dist/teachablemachine-image.min.js"></script>
 <script type="text/javascript">
-    // The json file defining the weights of the model
-    const checkpointURL = '{{MODEL_URL}}';
+    // the link to your model provided by Teachable Machine export panel
+    const modelURL = '{{MODEL_URL}}';
```

Comment:
I think this should be {{URL}} now. 

Currently there are 3 substitutions: 
MODEL_URL = `[...]/models/xyz/model.json`
CHECKPOINT_URL = `[...]/models/xyz/checkpoint.json`
URL = `[...]/models/xyz/`

Reply:
cool, will change.